ext/js/language/text-scanner.js

```diff
@@ -1139,30 +1122,18 @@ export class TextScanner extends EventDispatcher {
         this._scanTimerPromise = null;
     }
 
-    /**
-     * @returns {boolean}
-     */
-    _arePointerEventsSupported() {
-        return (this._pointerEventsEnabled && typeof PointerEvent !== 'undefined');
-    }
-
     /** */
     _hookEvents() {
         const capture = true;
         /** @type {import('event-listener-collection').AddEventListenerArgs[]} */
         let eventListenerInfos;
         if (this._searchOnClickOnly) {
             eventListenerInfos = this._getMouseClickOnlyEventListeners(capture);
-        } else if (this._arePointerEventsSupported()) {
-            eventListenerInfos = this._getPointerEventListeners(capture);
         } else {
             eventListenerInfos = [...this._getMouseEventListeners(capture)];
             if (this._scanWithoutMousemove) {
                 eventListenerInfos.push(...this._getKeyboardEventListeners(capture));
             }
-            if (this._touchInputEnabled) {
-                eventListenerInfos.push(...this._getTouchEventListeners(capture));
-            }
```

Comment:
We still need to call these functions to register the event listeners for touch and pen. Only need to remove the if check for `this._touchInputEnabled` and `this._pointerEventsEnabled` since we are removing these options and having them `true` by default.

Reply:
We might only want one or the other. According to #466, only pointer events should be required. Though I've never seen any browser that doesn't support both equally.

Reply:
Maybe need to test if this breaks touch behavior on popup components.

Reply:
So is the ask here to restore
```js
        } else if (this._arePointerEventsSupported()) {
            eventListenerInfos = this._getPointerEventListeners(capture);
``` 
?